traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java

```diff
@@ -755,7 +784,10 @@ public HTTPRouteResult singleRoute(final HTTPRequest request, final Track track)
 				track.setResultDetails(ResultDetails.DS_TLS_MISMATCH);
 				return null;
 			}
-			if (!ds.isAvailable()) {
+			if (ds.isRegionalGeoEnabled()) {
```

Comment:
I think this check was actually removed and the functionality fixed so probably shouldn't be added back in.

Reply:
done